final/implementation/esv-extensions.tex

```diff
@@ -1,10 +1,13 @@
 \section{ESV extensions}
 \label{sec:esv-extensions}
 
-To configure language specific features, such as the backend used to specify
-the dynamic semantics of a language (see \cref{sec:eval-strat}) and the start
-symbols valid within the context of a REPL, some REPL-specific extensions have
-been made available in the ESV language.
+REPL-specific extensions have been made available to the ESV language to
```

Comment:
Good one. I think this abbreviation needs to be introduced in chapter 1 though.

Also, currently a mix between ESV and esv is used. Spoofax itself uses esv, so we need to do so too, everywhere.